src/content/LandingPage/_landing-page.scss

```diff
@@ -0,0 +1,44 @@
+@import './mixins.scss';
+@import './overrides.scss';
+
```

Comment:
You're missing the `<landing-page__banner>` class. See https://www.carbondesignsystem.com/tutorial/react-step-2